src/python/pants/backend/codegen/protobuf/scala/rules.py

```diff
@@ -20,7 +20,7 @@
 from pants.core.util_rules.external_tool import DownloadedExternalTool, ExternalToolRequest
 from pants.core.util_rules.source_files import SourceFilesRequest
 from pants.core.util_rules.stripped_source_files import StrippedSourceFiles
-from pants.engine.environment import Environment, EnvironmentRequest
+from pants.engine.environment import EnvironmentVars, EnvironmentVarsRequest
```

Comment:
It would be good to move out of the module as well: perhaps to `pants.engine.env_vars` ...?

Reply:
(I like all of the class names though: thanks!)